test/samples/local/foundry_addr/contracts/test/Cheatcodes.t.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+import "../src/Counter.sol";
+
+contract CheatcodeTest is Test {
+    function setUp() public {
+    }
+
+    function testAddr() public {
+	    uint256 privKey = 0x6b911fd37cdf5c81d4c0adb1ab7fa822ed253ab0ad9aa18d77257c88b29b718e;
+	    // Correct address
+	    address addr = 0x16bB6031CBF3a12B899aB99D96B64b7bbD719705;
+	    // Incorrect address
+	    address badAddr = 0x16bb6031CbF3a12B899AB99d96b64b7BBD719706;
+
+	    address x = vm.addr(privKey);
+	    assertEq(x, badAddr);
```

Comment:
Same question as in https://github.com/ConsenSys/sol-dbg/pull/18#discussion_r1178627170.

Reply:
Just figured that we intentionally want test to fail. In that case - things are fine. Still, I would be grateful if ther would be a comment, that explicitly state our intentions (describe the case in words, if possible).

Reply:
Same as in other PR, leaving as is due to the effort to re-run the whole campaign due to one comment (source lines will mismatch if I just touch this file)